lib/json_api_resource/handlers/find_handler.rb

```diff
@@ -0,0 +1,19 @@
+module JsonApiResource
+  module Handlers
+    class FindHandler
+      attr_accessor :result
+
+      def initialize(results)
+        simplify!(results)
```

Comment:
do we need the bang in `simplify!`?  Its not raising errors, and its hard to say there are changes, because everything we do seems to have changes.